ui/pages/Validators.tsx

```diff
@@ -0,0 +1,158 @@
+import { Box, Hide, HStack, Show } from '@chakra-ui/react';
+import { useRouter } from 'next/router';
+import React from 'react';
+
+import { getFeaturePayload } from 'configs/app/features/types';
+import type { ValidatorsFilters, ValidatorsSorting, ValidatorsSortingField, ValidatorsSortingValue } from 'types/api/validators';
+
+import config from 'configs/app';
+// import useDebounce from 'lib/hooks/useDebounce';
+import useIsMobile from 'lib/hooks/useIsMobile';
+import { apos } from 'lib/html-entities';
+import getQueryParamString from 'lib/router/getQueryParamString';
+import { generateListStub } from 'stubs/utils';
+import { VALIDATOR } from 'stubs/validators';
+import ActionBar from 'ui/shared/ActionBar';
+import DataListDisplay from 'ui/shared/DataListDisplay';
+// import FilterInput from 'ui/shared/filters/FilterInput';
+import PageTitle from 'ui/shared/Page/PageTitle';
+import Pagination from 'ui/shared/pagination/Pagination';
+import useQueryWithPages from 'ui/shared/pagination/useQueryWithPages';
+import getSortParamsFromValue from 'ui/shared/sort/getSortParamsFromValue';
+import getSortValueFromQuery from 'ui/shared/sort/getSortValueFromQuery';
+import Sort from 'ui/shared/sort/Sort';
+import { SORT_OPTIONS } from 'ui/validators/utils';
+import ValidatorsCounters from 'ui/validators/ValidatorsCounters';
+import ValidatorsFilter from 'ui/validators/ValidatorsFilter';
+import ValidatorsList from 'ui/validators/ValidatorsList';
+import ValidatorsTable from 'ui/validators/ValidatorsTable';
+
+const Validators = () => {
+  const router = useRouter();
+  // const [ searchTerm, setSearchTerm ] = React.useState(getQueryParamString(router.query.address_hash) || undefined);
```

Comment:
will this be implemented later?

Reply:
yes, I hope so
this filter should be added in API in one of the next releases